resources/testdriver.js

```diff
@@ -657,31 +657,31 @@
          */
         in_automation: false,
 
-        click: function(element, coords) {
+        async click(element, coords) {
             if (this.in_automation) {
-                return Promise.reject(new Error('Not implemented'));
+                throw new Error("click() can't be called in automation");
```

Comment:
so what do I use instead, if I'm trying to click on something in automation?

this new error message would make me believe that I simply can't click on things in automation, would seems like a very common use-case

Reply:
ok, yeah, good point... Let me see if I can find some better wording or figure out why this is "not implemented".